### DISCUSSION TITLE
how can i use the real time line counter?

### DISCUSSION THREAD

**Discussion by StefanoGarzoni** [roboflow/notebooks#114] [Q&A] [answered]
hi, i have a project taken from the video : https://www.youtube.com/watch?v=OS5qI9YBkfk&t=198s
in which passing a video to the code, it processes a video result with the counting of the objects through a line counter, I would like to try to use this code but using my webcam (do the same things but in real time, I don't pass the video to the code), is it possible? and if so how can i do it?
I'm a beginner, but I'm getting very passionate about this world! Thanks in advance to anyone who can help me!

**Comment by SkalskiP** [comment, marked as answer]
Hi :) We have another video where we show tracking with YOLOv8 in real-time using the webcam.
- YouTube video: https://youtu.be/Mi9iHFd0_Bo
- related GitHub repository: https://github.com/SkalskiP/yolov8-native-tracking

**Reply by StefanoGarzoni** [reply]
once again, thank you very much!

**Reply by SkalskiP** [reply]
No problem :) happy to help if I have time
